### ISSUE TITLE
Copying a long value from debug watch results in pasting partial value

### ISSUE THREAD

**Issue by igiona** [rust-lang/rust-analyzer#15619]
**rust-analyzer version**: rust-analyzer version: 0.3.1657-standalone (326f37ef1 2023-09-10)

**rustc version**: rustc 1.72.0 (5680fa18f 2023-08-23)

**relevant settings**: none)

While in a debug session, copying a long string variable into the clipboard results in a truncated string:

```rust
 let auth_request_xml = format!("Looooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooong Striiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiinggggggggggggggggggggggg {}", username);
```
Copy Value results in:
```
"Loooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooong Striiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiinggggggggggggggggggggg...
```

![image](https://github.com/rust-lang/rust-analyzer/assets/1524038/9bf66348-97f3-459c-b1e4-b09b80d38530)

I would expect that the whole string is copied into the clipboard.

Apparently the C++ extension had a similar issue https://github.com/microsoft/vscode-cpptools/issues/5470 and it was reported in the old vscode extension already https://github.com/rust-lang/vscode-rust/issues/879

**Comment by lnicola** [comment]
We don't handle debugging, for that you need an extension like C++ Tools, Native Debug or CodeLLDB. You should check which one you're using and report it there.

**Comment by starball5** [comment]
Related: https://github.com/microsoft/vscode/issues/27950#issuecomment-613281366

**Comment by lnicola** [comment]
Nothing we can do about debugging, closing.
